common/scripts/build-uefi.sh

```diff
@@ -51,12 +51,13 @@
 TOP_DIR=`pwd`
 . $TOP_DIR/../../common/config/common_config.cfg
 
+# toolchain
+. $TOP_DIR/../../common/scripts/common_cross_toolchain.sh
```

Comment:
Please refactor the changes in file to a new file common/scripts/build-uefi-arm.sh

The original file common/scripts/build-uefi.sh shall be unmodified.